Launching-new-features.md

```diff
@@ -85,20 +91,12 @@ export enum FeatureNames {
 
 ### Modifying unit tests to account for the creation of the new flag
 
-The codebase contains two unit tests (one in the backend and one in the frontend) whose success or failure depends on them being aware of all the feature flags that currently exist.
-
-To make sure those two tests succeed, please follow the steps below (so they account for the newly created feature flag).
+The codebase contains two unit tests (one in the backend and one in the frontend) whose success or failure depends on them being aware of all the feature flags that currently exist. For the backend,
```

Comment:
I think you can replace all of this paragraph with "The above changes are enough to add a new feature flag to the backend."

Then you can start the next paragraph with: "However, to make sure the ..."

Reply:
Done